nilearn/glm/second_level/second_level.py

```diff
@@ -637,8 +637,8 @@ def compute_contrast(
 
         first_level_contrast : :obj:`str` or :class:`numpy.ndarray` of \
                             shape (n_col) with respect to \
-                            :class:`~nilearn.glm.first_level.FirstLevelModel`,
-                            default=None
+                            :class:`~nilearn.glm.first_level.FirstLevelModel` \
+                            or None, default=None
```

Comment:
Is the docstring here more elaborate than the previous ones? It considers the case `pandas.DataFrame` (a part from `FirstLevelModel`) being provided as `second_level_input`.

Reply:
hmmm... I need to add a test to make all cases for second level are covered: then if that's the case we should probably reuse a larger chunk of the docstring and try to indeed rely on _utils.doc

Reply:
added a test to make sure generate report also works when using a dataframe as input